### PR TITLE
add workflow to check if the files run

### LINKED ISSUE #11: Add github workflow to test the code
For instance, https://hsf-training.github.io/hsf-training-cicd-github/15-a-simple-test/index.html the workflow described here could be added with some modifications

### PR DESCRIPTION
**Pull request by amanmdesai**
Closes #11 
Tests #12 

The time taken by the worflow is ~8 minutes (see here: https://github.com/amanmdesai/hsf-training-cms-analysis/actions tested on my fork on a different branch)

The workflow only tests that the codes run. Does not perform any additional tests. 

Commands used: Workflow itself it mine (suggestions for enhancement/corrections are welcome)

The `run` commands come from the HSF CMS analysis webpage, this ensures that we also test all the commands stated on the webpage [HSF CMS Anylsis Webpage link](https://hsf-training.github.io/hsf-training-cms-analysis-webpage/index.html).

### CODE REVIEW

**Comment by klieret** [comment]
So if we settle on one ROOT version it should take less time anyway, right?

This is a really nice addition btw!

**Comment by amanmdesai** [comment]
> So if we settle on one ROOT version it should take less time anyway, right?
> 
> This is a really nice addition btw!

Thanks.  

No, the jobs run in a matrix (in parallel). So each takes (~8min) minutes.  The total workflow time is also the same (~8min), see e.g., https://github.com/amanmdesai/hsf-training-cms-analysis/actions/runs/3065087260/jobs/4948828221

**Comment by amanmdesai** [comment]
Main bottleneck is the skimming step as data is downloaded here (consumes almost 70% of the time). otherwise the analysis is pretty fast.

File: .github/test-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+     - master
+     - main
+
+jobs:
+  test-analysis:
+    runs-on: ubuntu-latest
+    container: rootproject/root:${{ matrix.version }}
+    strategy:
+      matrix:
+        version: [6.22.08-conda, 6.24.06-conda, latest]
```
Comment (klieret) [inline .github/test-analysis.yml]: I'd say lest still just keep `latest` in here to conserve resources

**Review by klieret — APPROVED** [review]

File: .github/test-analysis.yml (hunk 1 above)
Comment (amanmdesai) [reply .github/test-analysis.yml]: DONE

**Pull request merged**